docs/documentation/components/portal.mdx

```diff
@@ -2,7 +2,7 @@
 
 The Portal component is a utility component to help with the `ReactDOM.createPortal` API.
 Learn more about portal on the [official React docs.](https://reactjs.org/docs/portals.html) <br />
-Portals are used throughout Evergreen in components such as [Overlay](./overlay) and [Positioner](./positioner) to provide
+Portals are used throughout Evergreen in components such as [Overlay](./overlay.mdx) and [Positioner](./positioner.mdx) to provide
```

Comment:
Positioner link works fine as-is, but Overlay results in 404.

Reply:
`Overlay` is still a component in the v6 package but we missed it on our doc site. Will be opening a PR to address this.